symphony-bdk-examples/bdk-app-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -3,18 +3,31 @@ bdk:
   host: acme.symphony.com
   bot:
     username: mybot
-    privateKeyPath: /Users/user.name/.symphony/privatekey.pem
+    privateKeyPath: ${user.home}/.symphony/privatekey.pem
+
   app:
     appId: myapp
-    privateKeyPath: /Users/user.name/.symphony/privatekey.pem
+    privateKeyPath: ${user.home}/.symphony/privatekey.pem
 
 bdk-app:
   auth:
     enabled: true
     jwtCookie:
       enabled: true
-      maxAge: 1d
-
+      expireIn: 43200
```

Comment:
`maxAge` or `expireIn` (https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/297/files#diff-fe877abcd1c389c43d330bba208d249683706a06bba547a72fecc3e54796c3faR23) ? :)

Reply:
Oops! maxAge of course 😅